generator/internal/descriptor_utils.cc

```diff
@@ -836,6 +837,12 @@ VarsDictionary CreateServiceVars(
   vars["connection_header_path"] =
       absl::StrCat(vars["product_path"],
                    ServiceNameToFilePath(descriptor.name()), "_connection.h");
+  vars["connection_rest_cc_path"] = absl::StrCat(
+      vars["product_path"], ServiceNameToFilePath(descriptor.name()),
+      "_rest_connection.cc");
+  vars["connection_rest_header_path"] = absl::StrCat(
+      vars["product_path"], ServiceNameToFilePath(descriptor.name()),
+      "_rest_connection.h");
```

Comment:
are we concerned that a customer might see the rest connection header, but have no idea how to enable it?

(unlikely, as we know that Spanner Admin APIs are not used.... but this problem seems avoidable)